staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -96,8 +103,15 @@ func NewCustomResourceDefinitionHandler(
 		delegate:                delegate,
 		restOptionsGetter:       restOptionsGetter,
 		admission:               admission,
+		queue:                   workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "CustomResourceDefinitionHandler"),
 	}
 
+	crdInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    ret.addCustomResourceDefinition,
```

Comment:
Right, we don't need this.

_Sent from my Xiaomi Redmi Note 3 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_